src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -129,6 +128,16 @@ private async Task<bool> IsCentralPackageManagementVersionsEnabledAsync()
             return MSBuildStringUtility.IsTrue(await _vsProjectAdapter.GetPropertyValueAsync(ProjectBuildProperties.ManagePackageVersionsCentrally));
         }
 
+        private async Task<string> GetSpecifiedAssemblyName()
```

Comment:
Probably want to add async to the name.

Reply:
Merged without addressing this. I'll make make another PR later.